lib/themes/dosomething/paraneue_dosomething/js/components/ContextSlide.js

```diff
@@ -11,9 +11,21 @@ class ContextSlide extends React.Component {
 
   render() {
     return (
-      <div className="carousel__slide">
-        <h1>{"Woohoo! You're signed up."}</h1>
-        <p>here is some text</p>
+      <div className="container slideshow__slide">
```

Comment:
Having `slideshow__slide` here feels strange to me. Maybe this should be it's own BEM element, or the `slideshow__slide` should be applied as a wrapping container from the Slideshow component? (Not anything that's necessary to change for this PR, just thinking aloud).